test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndReaderGroupTest.java

```diff
@@ -204,6 +165,58 @@ public void testMultiScopeReaderGroup() throws Exception {
         assertTrue(managedStreams.contains(Stream.of(scopeB, streamName).getScopedName()));
     }
 
+    @Test(timeout = 30000)
+    public void getCurrentStreamCutTest() throws Exception {
```

Comment:
You are not consistently starting the test case name with `test`. Can we be consistent on the naming, please?

Reply:
updated the test method name.